.circleci/config.yml

```diff
@@ -98,9 +108,11 @@ jobs:
     environment:
       - CC: clang
       - CXX: clang++
+      - SDE_INSTALL_TAR: /tmp/bf-sde-tars/bf-sde-9.7.0-install.tgz
```

Comment:
I'm not sure if the resources spent on building with all SDE version justifies the small benefits. It's really only the sde wrapper and main.cc (which have no tests) that depend on it.

Reply:
S3 sync seems pretty fast (~3s), and the build should mostly be cached (negligible resources). So the most expensive thing here is probably extracting the tarball, which isn't easy to quantify without some additional logging.

Building the binary does force the linker to bring everything together which can uncover some issues.

I'd be curious to see the overhead before making the call not to do it.

Reply:
I agree, more tracing is needed. From our TST notes, we concluded to pursue and "main-only" approach at first. Let's solve this later once we have a few runs.